tests/e2e/page-objects/Documentation.js

```diff
@@ -0,0 +1,9 @@
+import 'testcafe';
+
+export default class Documentation {
+  constructor() {
+    this.URL = {
+      production: process.env.MP_URL || 'https://documentation.platform-os.com',
```

Comment:
Looks like you created class for documentation, so probably it would be better to just keep all the links regarding that in one place, here here

Example:
`documentation.URL.production +
      '/tutorials/user-uploads/uploading-files-directly-amazons3-using-ajax'`

this could become:
`this.article.userUploadsAjax`

Maintenance would be easier and readability in the test would be much higher, this concatenation in is hard to read.

Keep in mind you cant use "this.URL" inside the same object, so you would need to create this variable `this.article` next to this.URL

Reply:
Fixed